go-runtime/compile/schema.go

```diff
@@ -1278,6 +1433,95 @@ func (p *parseContext) pathEnclosingInterval(start, end token.Pos) (pkg *package
 	return nil, nil, false
 }
 
+func (p *parseContext) isPathInPkg(path string) bool {
+	if path == p.pkg.PkgPath {
+		return true
+	}
+	return strings.HasPrefix(path, p.pkg.PkgPath+"/")
+}
+
+func (p *parseContext) getEnumForTypeName(name string) (*schema.Enum, optional.Option[*types.Interface], bool) {
```

Comment:
Can you document this? I have no clue what the `bool` here does. If it's indicating whether the enum is being returned, just use an optional for that.

Reply:
Also, a valid alternative to comments in some circumstances is to name the return values. eg. if this was `enumPresent bool`